docs/aca/08-aca-monitoring/index.md

```diff
@@ -1,3 +1,7 @@
+---
+canonical_url: https://bitoftech.net/2022/09/22/azure-container-apps-auto-scaling-with-keda-part-11/
```

Comment:
this should be: https://bitoftech.net/2022/09/09/azure-container-apps-monitoring-and-observability-with-application-insights-part-8/